README.adoc

```diff
@@ -115,18 +115,19 @@ e.g. `-DBUILD_FORMATS=html`
 When only one build format is enabled the package name is transformed to include
 the format.
 
-==== SINGLE_LANGUAGE
+==== LANGUAGES
 
 By default CMake will configure to build all languages available for each document.
 
-You can build just a single language by using the **SINGLE_LANGUAGE** option when
-configuring a build with CMake, e.g. `-DSINGLE_LANGUAGE=it`, etc.
+You can build just a one or some of the languages by using the **LANGUAGES** option
+when configuring a build with CMake, e.g. `-DLANGUAGES=it;en;de`, etc.
 
-Currently, the available languages are : `en`, `fr`, `it`, `ja`, `nl`, and `pl` however, any
+Currently, the available languages are : `ca`, `de`, `es`, `fr`, `id`, `it`,
+`ja`, `nl`, `pl`, `ru`  and `zh`, however, any
 language code can be selected. Only translated documents will be built, so for
 some languages there may only be a partial documentation output.
 
-When the **SINGLE_LANGUAGE** option is set, the package name is transformed to
+When the **LANGUAGES** option is set, the package name is transformed to
```

Comment:
Should this be dropped, or should we concatenate the languages?

Reply:
What do you mean by dropped?

Reply:
I guess we can drop this. I don't think it even works that way right now anyways.